server/@types/bapv.d.ts

```diff
@@ -90,9 +90,9 @@ export type Restriction = {
   restrictionType: string
   restrictionTypeDescription: string
   startDate: string
-  expiryDate: string
+  expiryDate?: string
```

Comment:
Made these optional to match the API schema - and to placate TypeScript when I typed the test data.